maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/SurefireProperties.java

```diff
@@ -71,13 +71,22 @@ public SurefireProperties( KeyValueSource source )
         }
     }
 
+    @Override
+    public synchronized void putAll(Map<?, ?> t) {
+        for (Map.Entry<?, ?> entry : t.entrySet()) {
+            this.put(entry.getKey(), entry.getValue());
+        }
+    }
+
     @Override
     public synchronized Object put( Object key, Object value )
     {
         items.add( key );
         return super.put( key, value );
     }
 
+
+
```

Comment:
Here only one empty line between two methods. Feel free to remove these two you have added. Thx.